### PR TITLE
[ci.yaml] Update lsc script

### PR DESCRIPTION
**Pull request by CaseyHillers**
Moving this to be a local only as in practice, remote ci.yaml's are impractical to update (instead checkout a local copy and iterate on it).

Added some of the fields to the yaml now that they're being used

### CODE REVIEW

File: app_dart/bin/yaml_converter.dart [hunk 1]
```diff
@@ -63,27 +81,21 @@ void writeYaml(SchedulerConfig config) {
...
-  // There's an assumption that we're only generating builder configs from commits that
-  // have already landed with validation. Otherwise, this will fail.
+  final File ciYamlFile = File(args[0]);
+  final YamlMap configYaml = loadYaml(ciYamlFile.readAsStringSync()) as YamlMap;
   final SchedulerConfig schedulerConfig = schedulerConfigFromYaml(configYaml);
-  for (Target target in schedulerConfig.targets) {
-    target.name = target.builder;
-  }
+
+  //////////////////////////////////////////////////////////////////////////////
```
Comment (christopherfujino) [inline app_dart/bin/yaml_converter.dart:92]: nit. rather than the big comments, maybe a largeScaleChange helper function?

File: app_dart/bin/yaml_converter.dart [hunk 2]
```diff
@@ -30,6 +29,15 @@ void writeYaml(SchedulerConfig config) {
   for (Target target in config.targets) {
     configYaml.add('  - name: ${target.name}');
     configYaml.add('    builder: ${target.builder}');
+    if (target.enabledBranches.isNotEmpty) {
```
Comment (christopherfujino) [inline app_dart/bin/yaml_converter.dart:32]: how about adding a single test for this file that verifies the output? That way in code review we can easily preview the diff a change here would make to the generated output.

Comment (CaseyHillers) [reply app_dart/bin/yaml_converter.dart:49]: Good point, i'll add an integration test to validate an empty LSC returns the same Cocoon ci.yaml. I added a boiler plate, but it still needs some work
